Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:16.04
 RUN apt-get update
 RUN apt-get install -y python3-pip
 RUN pip3 install --upgrade pip
-RUN pip3 install tensorflow mysql-connector-python jupyter sqlflow
+RUN pip3 install tensorflow==2.0.0-alpha0 mysql-connector-python jupyter sqlflow
```

Comment:
What is the reason for switching to 2.0.0-alpha, for that we can use `keras.layers.DenseFeatures`? I noticed that ElasticDL is still using 1.13, which also has Eager Execution. We might need a comment there explaining the reason.

Reply:
Sure for causing the confusion. This PR should only contain the design doc.

TensorFlow 2.0.0-alpha uses `keras.layers.DenseFeatures`, while TensorFlow 1.13 uses `tf.feature_column.input_layer`, to transform feature columns to dense tensors. As mentioned [here](https://stackoverflow.com/a/54718798), Keras is becoming the de-facto high-level API for TensorFlow. So I would suggest using `keras.layers.DenseFeatures`, which is in TensorFlow 2.0.0-alpha.

Upgrading the Tensorflow version will cause failure in CI. I will remove this line change and file it in a separate PR.